packages/frontend-2/components/form/file-upload/ProgressRow.vue

```diff
@@ -1,21 +1,18 @@
 <template>
-  <div class="bg-foundation rounded-4xl px-4 py-3 w-full max-w-full relative">
+  <div
+    class="bg-foundation rounded-lg p-2 pr-1 w-full max-w-full relative dark:bg-foundation-2 bg-neutral-100"
```

Comment:
1. Use semantic colors whenever possible
2. The bg classes are clashing: there's both bg-foundation and bg-neutral-100

Reply:
Ah nice catch, that one was incidentally added, removed
 it